tests/test_sizing.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+
+import ast
+from vulture import Vulture
+from vulture.core import estimate_lines
+
+
+@pytest.fixture
+def wv():
+    return Vulture(verbose=True, sort_by_size=True)
+
+
+def test_size_function(wv):
+    wv.scan("""\
+def func():
+    if "foo" == "bar":
+        return "xyz"
+    import sys
+    return len(sys.argv)
+""")
+    assert wv.defined_funcs[0].size == 5
+
+
+def test_size_class(wv):
+    wv.scan("""\
+class Foo(object):
+    def bar():
+        pass
+
+    @staticmethod
+    def func():
+        if "foo" == "bar":
+            return "xyz"
+        import sys
+        return len(sys.argv)
+""")
+    assert wv.defined_classes[0].size == 9
```

Comment:
Can you use estimate_lines directly for the first two tests like for the third test, please?